script.js

```diff
@@ -13,4 +13,118 @@ closeImage.addEventListener('click', () => {
+}));
+
+const projectSection = document.querySelector('.work-cards');
+let render = '';
+
+const projectCard = [
+  {
+    title: 'Profesional Art Printing Data More',
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://ibtisam34.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ibtisam34/My-Portfolio',
+  },
+  {
+    title: 'Data Dashboard Healthcare',
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://ibtisam34.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ibtisam34/My-Portfolio',
+  },
+  {
+    title: 'Website Protfolio',
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://ibtisam.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ibtisam34/My-Portfolio',
+  },
+  {
+    title: 'Profesional Art Printing Data More',
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://ibtisam34.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ibtisam34/My-Portfolio',
+  },
+  {
+    title: 'Data Dashboard Healthcare',
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://ibtisam34.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ibtisam34/My-Portfolio',
+  },
+  {
+    title: 'Website Protfolio',
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://ibtisam34.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ibtisam34/My-Portfolio',
+  },
+];
+
+projectCard.forEach((data) => {
+  render += `<div id="w2" class="work1">
+  <div class="work1Discription" id="w2work1Discription">
+    <h2
+      <span id="w2d">${data.title}</span>
+    </h2>
+    <p>
+    ${data.description}
+    </p>
+    <ul class="category">
+      <li>${data.technology[0]}</li>
+      <li>${data.technology[1]}</li>
+      <li>${data.technology[2]}</li>
+    </ul>
+  </div>
+  <p class="title" hidden>${data.title}</p>
+  <p class="description" hidden>${data.description}</p>
+  <p class="livelink" hidden>${data.liveLink}</p>
+  <p class="sourcelink" hidden>${data.sourceLink}</p>
+  <a href="#" class="btn1" id="w2button">See Project</a>
```

Comment:
- [ ] The 'See project' button in the work cards should be a `button` tag that only opens the popup. Using the `a` tag is redirecting the user to the top of the page making a bad user experience. Kindly change that.